pkgs/by-name/pr/prism-model-checker/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  prism-model-checker-unwrapped,
+  jre,
+  symlinkJoin,
+  makeWrapper,
+  writeText,
+}:
+symlinkJoin {
```

Comment:
I don't see that much point in having a symlinkJoin with only one link in there — just wrap the programs in the "unwrapped" package (which should just become the main package)